ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh

```diff
@@ -74,7 +81,13 @@ EOF
 ping ${BASTION_IP} -c 10 || true
 echo "exit" | curl telnet://${BASTION_IP}:22 && echo "SSH port is opened"|| echo "status = $?"
 
-ansible-playbook -i $SHARED_DIR/bastion_inventory $SHARED_DIR/get-cluster-name.yml -vvvv
+CNF_ARG=""
```

Comment:
That's good for the first iteration. Please also check an option to manipulate actual parameters for upstream_cluster_all.py - for example:
```bash
ADDITIONAL_ARG=""
if [[ "$T5CI_JOB_TYPE" == "cnftests" ]]; then
  ADDITIONAL_ARG="--cluster-name cnfdc2 --force"
fi
```
And then just using the same task:

```yaml
- name: Discover cluster to run job
  command: python3 ~/telco5g-lab-deployment/scripts/upstream_cluster_all.py --get-cluster -e $CL_SEARCH $ADDITIONAL_ARG
  register: cluster
  environment:
    JOB_NAME: ${JOB_NAME:-'unknown'}
```
Then we leave ansible execution as is and changing just the params inside.